jobs/build/promote/Jenkinsfile

```diff
@@ -744,6 +744,7 @@ node {
                         build(
                                 job: '/aos-cd-builds/build%2Fcincinnati-prs',  propagate: true,
                                 parameters: [
+                                    buildlib.param('String', 'FROM_RELEASE_TAG', params.FROM_RELEASE_TAG),
```

Comment:
this will be empty in the case of promoting an assembly

the assembly may have a nightly defined (in `from_release_tag`) or may not

Reply:
ok, no problemo. i'll handle the empty value inside the function

Reply:
done in https://github.com/openshift/aos-cd-jobs/pull/2768/commits/34d5e86c9ce107dfe8f5e3b82ddd775cbd7558a9